docs/pages/management/security.mdx

```diff
@@ -1,11 +1,26 @@
 ---
-title: Security
-description: Hardening the security of your Teleport deployment
-layout: tocless-doc
+title: Secure cluster operations
```

Comment:
```suggestion
title: Harden your Teleport cluster
```
Suggested to make this more specific:
- It's advice for making your Teleport cluster more secure, not operating securely within a cluster
- It makes it more explicit that we're referring to Teleport clusters for users arriving from Google